components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -544,10 +571,49 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
         }
     }
 
+    fn should_reschedule(&self, written_bytes: u64) -> bool {
+        let control = self.apply_flow_control();
+        written_bytes >= control.yield_written_bytes
+            || control.timer.saturating_elapsed() >= control.yield_time
+    }
+
+    pub async fn maybe_reschedule(&mut self, written_bytes: u64) {
```

Comment:
Why it only checks the written bytes of one single `flush`?

Reply:
The original implementation only checks diff in the end of every round. https://github.com/tikv/tikv/pull/13594. I think check accumulated bytes will hurt performance as the configuration is several KiBs.